### ISSUE TITLE
Profile Component Accessibility Issue: Quote flagging "overuse of italics" in SiteImprove

### ISSUE THREAD

**Issue by bs005600** [wvuweb/wvu-design-system-v2#94]
The Profile component's quote is flagging an overuse of italics accessibility issue in SiteImprove. The "iowan-old-style-black-italic" class can be moved off of the div (line 119) and onto an inner span tag.

**Component on Github:**
(https://github.com/wvuweb/wvu-design-system-v2/blob/master/views/components/wvu-profile/_wvu-profile--v1.html).

**Quote Code:**
Starts on Line 117 and ends on Line 122. 

**SiteImprove Example:** 
(https://my2.siteimprove.com/Inspector/55860/A11Y/Page?pageId=73897116288&impmd=55BEE6BA68D67FE07C317E34EB77BDF8&conf=a+aa+aria+si#/sia-r85/failed/TjYyR2ZxbFk=;ZGY2ZDU0Nzc=).
